src/components/PeopleLink.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../types';
+import cn from 'classnames';
+
+type Props = {
+  person: Person;
+};
+
+export const PeopleLink: React.FC<Props> = ({ person }) => {
+  const {
+    name,
+    sex,
+    born,
+    died,
+    fatherName,
+    motherName,
+    slug,
+    mother,
+    father,
+  } = person;
+  const { slug: personId } = useParams();
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({
+        'has-background-warning': personId === slug,
+      })}
+    >
+      <td>
+        <Link
+          to={`/people/${slug}`}
+          className={cn({
+            'has-text-danger': sex === 'f',
```

Comment:
The conditional class 'has-text-danger' is added if 'sex' is 'f'. However, it might be more intuitive to name your classes based on their function rather than their condition. For example, you could name this class 'female' instead of 'has-text-danger'. This would make your code more readable and maintainable.